src/Shared/CommunicationsUtilities.cs

```diff
@@ -608,7 +608,7 @@ internal static HandshakeOptions GetHandshakeOptions(bool taskHost, string archi
             switch (clrVersion)
             {
                 case 0:
-                    // Not a taskhost, runtime must match
+                // Not a taskhost, runtime must match
```

Comment:
Why unindent this?

Reply:
It was automatic formatting change. I don't know why the formatter did that just for this case, but I reverted it.